content/docs/getting-started/hello-world.md

```diff
@@ -204,8 +204,11 @@ You can find the full example [here][full-code].
 command starts a listening TCP socket on the previously specified port.
 
 ```bash
-$ nc -l 6142
+$ nc -l -p 6142
 ```
+> The command above is used with the GNU version of netcat that comes stock on many
+> unix based operating systems. The following command can be used with the
+> [NMap.org][NMap.org] version: `$ ncat -l -p 6142`
```

Comment:
This doesn't make sense to me.

The default example and the "nmap.org version" example as the same thing.

Reply:
one is `nc ...` and one is `ncat`. If it is more confusing with the same arguments then maybe something like "if using nmap org version, substitute ncat for nc" or something like that?

Reply:
Ah, gotcha.  I was focused on the arguments, not the binary name. :)